runelite-client/src/main/java/net/runelite/client/plugins/slayer/SlayerPlugin.java

```diff
@@ -172,6 +172,28 @@
 	private Instant infoTimer;
 	private boolean loginFlag;
 
+	/**
+	 * Get npc composition, account for imposters
+	 *
+	 * @param npc
+	 * @return
+	 */
+	private static NPCComposition getComposition(NPC npc)
```

Comment:
Why is this function moved?

Reply:
Whoops my auto-format apparently was set up to do this. Reverted to how it was before + changed my settings.